### ISSUE TITLE
Not able to see logs in argo cd

### ISSUE THREAD

**Issue by poojaambali** [argoproj/argo-cd#20321]
Checklist:

* [ ] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [ ] I've included steps to reproduce the bug.
* [ ] I've pasted the output of `argocd version`.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Version**

```shell
Paste the output from `argocd version` here.
```

**Logs**

```
Paste any relevant application logs here.
```

**Comment by poojaambali** [comment]
<img width="960" alt="image (1)" src="https://github.com/user-attachments/assets/7fb11dab-a2ad-4ea5-8cab-ec42360ce753">

**Comment by linghaoSu** [comment]
Which version of ArgoCD are you using?

This might be the same issue.
https://github.com/argoproj/argo-cd/issues/20132

**Comment by poojaambali** [comment]
v2.14.0+04e47cd

**Comment by linghaoSu** [comment]
It seems the version you're using is earlier than the fix. Could you try using a dev version built after commit 20e2e784764bf4907fd29f83d34666016b73dfc8?

![image](https://github.com/user-attachments/assets/b2ddfea2-b9a1-4883-9ba4-4780d7584318)

**Comment by nitishfy** [comment]
Try running [argocd locally](https://argo-cd.readthedocs.io/en/stable/developer-guide/running-locally/) using `make start-local` command and see If you're getting that error.

**Comment by agaudreault** [comment]
Fixed by #20133
